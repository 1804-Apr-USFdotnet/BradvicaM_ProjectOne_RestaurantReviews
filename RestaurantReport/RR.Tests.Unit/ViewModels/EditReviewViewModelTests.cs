using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using RR.ViewModels;

namespace RR.Tests.Unit.ViewModels
{
    [TestClass]
    public class EditReviewViewModelTests
    {
        [TestMethod]
        [UseReporter(typeof(DiffReporter))]
        public void ToString_OnCall_ReturnsCorrectFormat()
        {
            var viewModel = new EditReviewViewModel
            {
                Comment = "Some Comment",
                Rating = 5.5,
                ReviewerName = "Some Name",
                ReviewPublicId = Guid.Empty
            };

            Approvals.Verify(viewModel);
        }
    }
}
