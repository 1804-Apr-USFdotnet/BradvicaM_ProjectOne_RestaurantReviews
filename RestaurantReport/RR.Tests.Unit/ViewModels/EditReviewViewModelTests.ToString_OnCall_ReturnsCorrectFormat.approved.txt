
PublicId: 00000000-0000-0000-0000-000000000000
ReviewerName: Some Name
Rating: 5.5
Comment: Some Comment
